src/main/java/hu/oe/nik/szfmv/automatedcar/model/AutomatedCar.java

```diff
@@ -1,12 +1,13 @@
-package hu.oe.nik.szfmv.automatedcar;
+package hu.oe.nik.szfmv.automatedcar.model;
```

Comment:
ne a modelbe dobálgassuk ezeket, szerintem legyen hu.oe.nik.szfmv.automatedcar.model.objects package (vagy valami hasonló vagy beszédesebb névvel és oda) a főpackaget tartsuk fent kicsit globálisabb dolgokra, mint a transform meg position amit nem lehet egyértelműen kategorizálni és sok alpackeben is használjuk őket

Reply:
Szóval azok az objektumok, amiket példányosítunk az alkalmazás során, kerüljenek át a model.objectsbe, amik pedig segéd osztályok, azok legyenek a modelben? Fordítva még elmenne, de szerintem az lenne legszebb, ha átkerülhetne pl egy model.dto packagebe a Position és a Transform.

Reply:
Ok akkor a package marad